### ISSUE TITLE
[bug] Apktool version detection

### ISSUE THREAD

**Issue by Fisjkars** [sensepost/objection#364]
**Describe the bug**
Issue in the version detector for apktool on **Windows 10**

**To Reproduce**
objection patchapk -s .\base.apk

**Evidence / Logs / Screenshots**
The stacktrace I had :
```
objection patchapk -s .\base.apk
No architecture specified. Determining it using `adb`...
Detected target device architecture as: arm64-v8a
Using latest Github gadget version: 12.8.20
Patcher will be using Gadget version: 12.8.20
Detected apktool version as: 2.4.1
Appuyez sur une touche pour continuer...
apktool version should be at least 2.4.1
Please see the following URL for more information: https://github.com/sensepost/objection/wiki/Apktool-Upgrades
apktool is not ready for use
Cleaning up temp files...
Failed to cleanup with error: [WinError 2] Le fichier spécifié est introuvable: 'C:\\Users\\blah\\AppData\\Local\\Temp\\tmp51gd96lf.apktemp.objection.apk'

```
```text
objection --debug explore
```

**Environment (please complete the following information):**
 - Device: Galaxy Tab s3
 - OS: Windows 10
 - Frida NA
- Objection Version objection: 1.9.1

**Comment by leonjza** [comment]
Unfortunately Windows support is limited. PR's welcome to fix it though.

**Comment by LouisBellinger** [comment]
I'm also facing the issue. Quite frustrating.

**Comment by leonjza** [comment]
> I'm also facing the issue. Quite frustrating.

Happy to take a PR to improve this.

**Comment by LouisBellinger** [comment]
Alright so this is related to [this other issue](https://github.com/sensepost/objection/issues/345). In \objection\utils\patchers\android.py line 228:

```
# windows / delegator weirdness...
if 'Press any key to continue . . .' in o:
    o = o.split('\n')[0]
```
The 'Press any key to continue . . .' check won't work if your os language is not in english. For instance, if you're using French, the check should be:

`if 'Appuyez sur une touche pour continuer...' in o:`

**Comment by leonjza** [comment]
I just released 1.9.2 with a more generic approach to this Windows weirdness.
